site/docs/master/sonobuoy-config.md

```diff
@@ -54,14 +54,45 @@ Filters
    - A Kubernetes [label selector][labelselector] which will be added to every query run.
 
 Limits
- - Options for limiting limits the scope of response when getting logs from pods. These will be passed onto Kubernetes [PodLogOptions][podlogopts]
- - The supported parameters are:
-    - Previous
-    - SinceSeconds
-    - SinceTime
-    - Timestamps
-    - TailLines
-    - LimitBytes
+ - Options for limiting limits the scope of response.
+ - **Limits.PodLogs** limits the scope when getting logs from pods. 
+     - The scope of pods is defined by:
+       ```
+       Namespaces OR SonobuoyNamespace OR FieldSelectors) AND LabelSelector
+       ```
+     - The supported parameters are:
+        - **Namespaces**: string
+            - A regular expression for the targeted namespaces, following the format of Golang Regexp. 
```

Comment:
nit: I'd remove `following the format of Golang Regexp`. There are a few things a golang regexp cant do but I don't think they really apply much here and can just confuse the user into thinking `golang regexp` is something different than normal regexp.

Reply:
okay, just trying to say that goland use something like ".*" while other language maybe use "*"